dev/com.ibm.websphere.appserver.sessionCache-1.0/resources/l10n/com.ibm.websphere.appserver.sessionCache-1.0.properties

```diff
@@ -14,4 +14,7 @@
 #NLS_MESSAGEFORMAT_NONE
 #
 
-description=TODO write useful description for this feature before we beta it
+description=This feature enables persistence of HTTP sessions using JCache. Persisting \
+ HTTP session data using JCache allows for high performance HTTP session persistence \
+ without the use of a relational database.  Failover of HTTP sessions can be achieved \
```

Comment:
Make sure there is only one space after relational database.

Reply:
I'll get that updated.